tests/pkg_relic/Makefile

```diff
@@ -0,0 +1,46 @@
+include ../Makefile.tests_common
+
+# The following boards are known to fail or have not been tested.
+BOARD_BLACKLIST :=  arduino-duemilanove \
+                    arduino-mega2560 \
+                    arduino-uno \
+                    chronos \
+                    f4vi1 \
+                    jiminy-mega256rfr2 \
+                    mega-xplained \
+                    msb-430 \
+                    msb-430h \
+                    msbiot \
+                    redbee-econotag \
+                    stm32f0discovery \
+                    stm32f3discovery \
+                    telosb \
+                    waspmote-pro \
+                    wsn430-v1_3b \
+                    wsn430-v1_4 \
+                    z1 \
+                    #
+
+BOARD_INSUFFICIENT_MEMORY :=    nucleo-f030r8 \
+                                nucleo-f031k6 \
+                                nucleo-f042k6 \
+                                nucleo-l031k6 \
+                                nucleo-l053r8 \
+                                #
+
+TEST_BOARDS_LLVM_COMPILE := native \
```

Comment:
This should be done in the `relic` package to blacklist llvm if it is not compatible using `TOOLCHAIN_BLACKLIST += llvm` with the correct conditionals.